docusaurus/docs/iOS/uikit/data-formatting.md

```diff
@@ -54,4 +54,25 @@ Appearance.default.formatters.userLastActivity = CustomUserLastActivityFormatter
 | ------------- | ------------- |
 | <img src={require("../assets/data-formatting-custom-user-activity-before.png").default} /> | <img src={require("../assets/data-formatting-custom-user-activity-after.png").default} /> |
 
-With this custom formatter we changed the last activity to display the relative date and the exact time which the user was last seen.
\ No newline at end of file
+With this custom formatter we changed the last activity to display the relative date and the exact time which the user was last seen.
+
+## Changing the channel name formatting
+You can also create a custom formatter that uses a combination of our default formatters with your custom logic. For example, you can create a custom `ChannelNameFormatter` that only uses custom logic when the member count is only 1, but for the rest of the cases it uses the default logic.
+
+```swift
+class CustomChannelNameFormatter: ChannelNameFormatter {
+    let defaultChannelNameFormatter = DefaultChannelNameFormatter()
+
+    func format(channel: ChatChannel, forCurrentUserId currentUserId: UserId?) -> String? {
+        if channel.memberCount == 1 {
+            return channel.membership?.name
+        }
+
+        return defaultChannelNameFormatter.format(channel: channel, forCurrentUserId: currentUserId)
+    }
+}
+
+Appearance.default.formatters.channelName = CustomChannelNameFormatter()
+```
+
+In the example above, if the channel has only the current user as a member, it uses the name of the current user as the channel name, otherwise the default logic is used.
```

Comment:
```suggestion
In the example above, if the channel has only the current user as a member, it uses the name of the current user as the channel name, otherwise, the default logic is used.
```